### PR TITLE
{bio}[foss/2020a] CMSeq v1.0.3 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by manifestoso**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by manifestoso** [comment]
Test report by @manifestoso
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
hpc-login01 - Linux RHEL 7.5, x86_64, AMD EPYC 7551 32-Core Processor, Python 3.6.10
See https://gist.github.com/8ad7d9d630d811aab22e4d26072ee337 for a full test report.

**Comment by manifestoso** [comment]
Test report by @manifestoso
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
hpc-login01 - Linux RHEL 7.5, x86_64, AMD EPYC 7551 32-Core Processor, Python 3.6.10
See https://gist.github.com/573e6289f46a0d72579eea931c20b19a for a full test report.

**Comment by manifestoso** [comment]
Test report by @manifestoso
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
hpc-login01 - Linux RHEL 7.5, x86_64, AMD EPYC 7551 32-Core Processor, Python 3.6.10
See https://gist.github.com/af3d807b1d4b3f465da053cb71cb90ba for a full test report.

**Comment by manifestoso** [comment]
depends on PR#14321 
@verdurin LGTM.

**Comment by manifestoso** [comment]
LGTM

File: easybuild/easyconfigs/c/CMSeq/CMSeq-1.0.3-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+checksums = ['c354b89bc4fdbb24fa1ffa063febdb96f2bbd8837af399887b300c9be98bda2e']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('Biopython', '1.78', versionsuffix),
+    ('bcbio-gff', '0.6.6', versionsuffix),
+    ('biom-format', '2.1.10', versionsuffix),
+    ('Pysam', '0.16.0.1'),
+    ('SAMtools', '1.12', '', ('GCC', '9.3.0')),
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CMSeq/CMSeq-1.0.3-foss-2020a-Python-3.8.2.eb:29]: @manifestoso please check whether you can use `SAMtools` `1.10` - see the unit test error for details.

**Review by verdurin — DISMISSED** [review]

File: easybuild/easyconfigs/c/CMSeq/CMSeq-1.0.3-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (manifestoso) [reply easybuild/easyconfigs/c/CMSeq/CMSeq-1.0.3-foss-2020a-Python-3.8.2.eb]: @verdurin fixed, thx.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3508.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a4b13d5a596c58935fed42a37dbdb857 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3105.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/afc468c18253958bb50ab59a41f13361 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @manifestoso!

**Pull request merged**